app/coffee/Router.coffee

```diff
@@ -77,6 +78,7 @@ module.exports = Router =
 
 			client.on "disconnect", () ->
 				metrics.inc('socket-io.disconnect')
+				metrics.gauge('socket-io.clients', io.sockets.clients()?.length - 1)
```

Comment:
I would rather place it after the leaveProject call.

Consider a case with two connected clients. Now both are disconnecting at the same time. The metric will likely remain at 1.